pkg/orchestration/wiring/testdata/saml.label.state.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: orchestration.voyager.atl-paas.net/v1
+kind: State
+metadata:
+  name: state1
+  namespace: default123
+  uid: 411c0040-617e-11e7-9b57-427d691976d7
+  labels:
+    voyager.atl-paas.net/label: my-label
+spec:
+  resources:
+  - name: saml
+    type: SAML
+    spec:
+      url: 'https://my-service.prod'
+  - name: kubecompute-simple
+    type: KubeCompute
+    dependsOn:
+    - saml
+    spec:
+      containers:
+      - name: microservice
+        image: "docker.example.com/atlassian/micros-analytics:0.1"
```

Comment:
nit: this is a weird example: image has a tag, and there is a separate tag below. the correct format is not to have a tag in the `image` IIRC.

Reply:
but given that this is a SAML test, this is not critical at all.